Assets/Scripts/AIForNPC/AIForNPC.cs

```diff
@@ -19,6 +19,8 @@ public class AIForNPC : MonoBehaviour
     private float previousTime = 0f;
     private int currentSeed = 0;
 
+    public int id;
```

Comment:
If id has to be public: add [HideInInspector] attribute and move to part with public variables.
If id has to be private: add private modifier